### DISCUSSION TITLE
When will version 5.5 be updated?

### DISCUSSION THREAD

**Discussion by siztankc** [pafuhana1213/KawaiiPhysics#147] [Q&A] [answered]
First of all, I feel very grateful to you. After updating the engine to 5.5, various errors have occurred. Could you let me know when the 5.5 update will be available?

**Comment by lordrex34** [comment]
https://github.com/pafuhana1213/KawaiiPhysics/issues/148

this might help, I did a quick fix for myself and shared

**Comment by pafuhana1213** [comment, marked as answer]
Hi guys!
Sorry for my late support... I've released UE5.5 support now!
please check it! thanks!
https://github.com/pafuhana1213/KawaiiPhysics/discussions/150
